src/query/procedure/module.hpp

```diff
@@ -117,6 +117,8 @@ class ModuleRegistry final {
   /// Returns the shared memory allocator used by modules
   utils::MemoryResource &GetSharedMemoryResource() noexcept;
 
+  bool RegisterProcedure(const std::string_view name, mgp_proc proc);
```

Comment:
I would rename this to `RegisterBuiltinProcedure`, because this function cannot be used to register arbitrary procedures, only the "builtin" ones to the `mg` module.

Reply:
Good suggestion!